conda/recipes/pylibcugraph/meta.yaml

```diff
@@ -58,6 +58,8 @@ requirements:
     - libcugraph ={{ version }}
     - pylibraft ={{ minor_version }}
     - python
+    # TODO: Remove rmm once https://github.com/rapidsai/raft/pull/1656 is merged (fixes pylibraft use of rmm)
+    - rmm ={{ minor_version }}
```

Comment:
Are there any linkages to RMM from `pylibcugraph` itself? Does it use `rmm`'s Cython interface?